routing/validation_barrier.go

```diff
@@ -201,24 +168,7 @@ func (v *ValidationBarrier) WaitForDependants(job interface{}) error {
 		jobDesc = fmt.Sprintf("job=channeldb.LightningNode, pub=%x",
 			vertex)
 
-	case *lnwire.ChannelUpdate:
-		signals, ok = v.chanEdgeDependencies[msg.ShortChannelID]
-
-		jobDesc = fmt.Sprintf("job=lnwire.ChannelUpdate, scid=%v",
-			msg.ShortChannelID.ToUint64())
-
-	case *lnwire.NodeAnnouncement:
```

Comment:
Ah yeah, since we now have distinct validation barriers for the router vs the gossiper? Good move.

Reply:
Yep. I think eventually it will be removed from the router so we have a clear workflow here, `msgs -> gossiper -> router`, where the gossiper handles resource limiting/caching/validating messages, and the router handles saving them to db. The final change should be straightforward, that we unify the message flow in gossiper, making the policy updates also go to  `networkMsgs`, then the router's vb can be removed.